habitat-hitl/habitat_hitl/_internal/networking/keyframe_utils.py

```diff
@@ -95,18 +95,12 @@ def ensure_dict(keyframe, key):
 
     if "message" in inc_keyframe:
         inc_message = inc_keyframe["message"]
-        # add/update these messages
-        for message_key in [
-            "teleportAvatarBasePosition",
-            "sceneChanged",
-            "navmeshVertices",
-            "isAppReady",
-        ]:
-            if message_key in inc_message:
-                ensure_dict(consolidated_keyframe, "message")
-                consolidated_keyframe["message"][message_key] = inc_message[
-                    message_key
-                ]
+        # add/update all messages
+        for message_key in inc_message:
```

Comment:
It seems like a good general rule that consolidating messages simply means updating to the latest message in the latest incremental keyframe. Let's beware a future situation where a client must absolutely receive an entire sequence of messages (not just the latest one).